File: react/src/main/java/com/react/model/course/Course.java
package com.react.model.course;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Entity
@Table(name = "courses")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Course {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false)
    private String title;

    @Column(columnDefinition = "TEXT")
    private String description;

    private Double price = 0.0;

    @Column(name = "instructor_id")
    private String instructorId;

    @Column(nullable = false)
    private LocalDateTime createdAt = LocalDateTime.now();

    // getters & setters
}

File: BaseSpringSecurity/src/main/kotlin/entity/Course.kt
package com.entity

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.*

@Entity
@Table(name = "course")
data class Course (

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Int? = null,
    var name: String? = null,
    var description: String? = null,
    var active: Boolean? = null,
    @Column(name = "url_img")
    var urlImg: String? = null
)